modules/gitops-installing-argo-rollouts-cli-using-linux.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-linux_{context}"]
+= Installing Argo Rollouts CLI using Linux
+
+For Linux, you can install Argo Rollouts CLI by performing the following steps.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (oc).
+
+.Procedure
+
+. Download the latest version of the Argo Rollouts CLI binary, `kubectl-argo-rollouts`, by running the following command using `curl`.
++
+.Example - Downloading the Argo Rollouts CLI binary
+[source,yaml]
+----
+curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+----
+
+. Ensure that the `kubectl-argo-rollouts` binary is executable by running the following command.
++
+.Example - Marking the `kubectl-argo-rollouts` binary as executable
+[source,yaml]
+----
+chmod +x ./kubectl-argo-rollouts-linux-amd64
+----
+
+. Move the `kubectl-argo-rollouts` binary to the system PATH by running the following command.
++
+.Example - Moving the `kubectl-argo-rollouts` binary to the system path
+[source,yaml]
+----
+sudo mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
+----
+
+The Argo Rollouts CLI binary is installed.
+
+.Verification
+
+After you have completed the installation of the Argo Rollouts CLI plugin, verify the installation of the Argo Rollouts CLI plugin by running the following command:
+
+.Example - Verifying the Argo Rollouts version
+[source,yaml]
+----
+oc argo rollouts version
+----
+
+As a result, the following output is displayed:
+
+.Example - Output of the Argo Rollouts version command
+[source,yaml]
+----
+kubectl-argo-rollouts: v1.6.6+737ca89
+  BuildDate: 2024-02-13T15:39:31Z
+  GitCommit: 737ca89b42e4791e96e05b438c2b8540737a2a1a
+  GitTreeState: clean
+  GoVersion: go1.20.14
+  Compiler: gc
+  Platform: linux/amd64
```

Comment:
how will the user determine that the installation worked correctly? Is there anything to look for in this output? Add callouts.

Reply:
As per my interaction with the SME, the installation is completed after the 3rd step in this procedure. The verification steps are added to check the version of the downloaded binary package.

Reply:
Verification steps will also verify that the installation was completed correctly.